reana_commons/openapi_specifications/reana_server.json

```diff
@@ -2386,6 +2386,7 @@
                     }
                   }
                 },
+                "reana_server_version": "0.8.0",
```

Comment:
This shouldn't necessarily change since it is just docs (see in [`maint-0.7`](https://github.com/reanahub/reana-commons/blob/1f0ffc6b9de05163e9f94d203a0692a17d120577/reana_commons/openapi_specifications/reana_server.json#L2469)). I guess the bump dependencies script just upgraded the version automatically (through grep).

While checking this I realised that, although it is not having an impact code wise, we have a mistake in R-Commons `0.7.1` release. I've bumped RS, RWE[Yadage/CWL/Serial] and RJC so their openapi specs got a version upgrade, but I forgot to push that change back to REANA-Commons, so [it still shows as a dev version](https://github.com/reanahub/reana-commons/blob/1f0ffc6b9de05163e9f94d203a0692a17d120577/reana_commons/openapi_specifications/reana_server.json#L6).

Reply:
Yes, I wanted to change the docs too, in case anybody is reading them as "live" example, so that we are consistent across maint-X.Y families.

Something we can streamline when doing upgrade of shared modules and/or release tagging later on...